code/WorkInProgress/pomf/spacepods/spacepods.dm

```diff
@@ -216,15 +216,15 @@
 		if("pod_civ")
 			desc = "A sleek civilian space pod."
 		if("pod_black")
-			desc = "An all black space pod with no insignias."
+			desc = "A plain black space pod without any distinctive markings."
 		if("pod_mil")
-			desc = "A dark grey space pod brandishing the Nanotrasen Military insignia"
```

Comment:
I've never heard "Nanotrasen Military" used before, so I think it's the latter. "Nanotrasen Enforcement and Protection Division" or something shorter, perhaps?